pom.xml

```diff
@@ -195,6 +200,14 @@
         <plugin>
           <artifactId>maven-surefire-plugin</artifactId>
           <version>2.18.1</version>
+          <dependencies>
+            <dependency>
+              <!-- Force junit provider because surefire will choose the testng one if testng is detected -->
+              <groupId>org.apache.maven.surefire</groupId>
+              <artifactId>surefire-junit47</artifactId>
```

Comment:
@juherr 
Strange because I have updated the plugin section only and I used Maven 3.2.1:
`<plugin>
                <artifactId>maven-surefire-plugin</artifactId>
                <version>2.18.1</version>
                <configuration>
					<threadCount>1</threadCount>
                    <properties>
                        <property>
                            <name>junit</name>
                            <value>false</value>
                        </property>
                    </properties>
                </configuration>
                <dependencies>
                    <dependency>
                        <groupId>org.apache.maven.surefire</groupId>
                        <artifactId>surefire-junit47</artifactId>
                        <version>2.18.1</version>
                    </dependency>
                    <dependency>
                        <groupId>org.apache.maven.surefire</groupId>
                        <artifactId>surefire-testng</artifactId>
                        <version>2.18.1</version>
                    </dependency>
                </dependencies>
            </plugin>`